.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  extends: "@ethereumjs/eslint-config-defaults",
+  parserOptions: {
+    project: ['./tsconfig.json', './tsconfig.browser.json', './tsconfig.eslint.json']
+  },
+  rules: {
+    // Many methods have been sketched in as stubs & their params trigger this.
+    // Duplicates the (more tolerant) @typescript-eslint/no-unused-vars
+    'no-unused-vars': 'off'
+  }
+}
```

Comment:
Small note about `parserOptions`... was getting an error complaining of files unreachable through the tsconfig similar to:
```
error: Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: unusedFile.ts.
The file must be included in at least one of the projects provided at unusedFile.ts:
```
Have followed the suggestions at [typescript-eslint#parserOptions.project][1] to add the `browser` folder files and `webpack.config.js` to the lint set.

[1]: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/parser/README.md#parseroptionsproject